compiler/modules/nback.llvm/llvm.bal

```diff
@@ -128,8 +131,21 @@ public class Module {
             "smul.with.overflow.i64" => {
                 fn = new ("llvm.smul.with.overflow.i64", overflowArithmeticFunctionType);
             }
+            "ptrmask.p0i8.i64" => {
+                FunctionType fnType = {returnType: pointerType("i8"), paramTypes:[pointerType("i8"),"i64"] };
+                FunctionDecl f = new("llvm.ptrmask", fnType);
```

Comment:
I used "llvm.ptrmask" as the function name here because the llvm langue reference don't indicate type specific overloads (ref:https://llvm.org/docs/LangRef.html#llvm-ptrmask-intrinsic).

Reply:
The overloading stuff is explained here https://llvm.org/docs/LangRef.html#intrinsic-functions

But it's not very clear. I had to experiment to figure it out:

https://github.com/ballerina-platform/nballerina/blob/main/experiment/taggedptr/taggedptr.ll

I had to look at the LLVM test cases to figure `p0i8` is supposed to be in the name for `i8*`.